dashboard/app/models/stage.rb

```diff
@@ -136,13 +136,13 @@ def lockable_state(students)
     end
     script_level = self.script_levels[0]
     return students.map do |student|
-      user_level = UserLevel.find_by(user_id: student.id, level: script_level.level, script_id: self.script.id)
+      user_level = student.last_attempt_for_any script_level.levels, script_id: self.script.id
```

Comment:
@ashercodeorg This will ultimately call (see user.rb below):
```
UserLevel.where(
    user_id: student.id, 
    level_id: script_level.levels.map(&:id), 
    script_id: self.script.id)
  .order('updated_at DESC')
  .first
```

So the query now takes multiple (usually just 1, occasionally 2) level ids instead of 1, and includes a new `order by` clause. Doesn't seem like that big of a change to me.